x-pack/plugins/cases/common/types/domain/index.ts

```diff
@@ -7,6 +7,8 @@
 
 // Latest
 export * from './configure/latest';
+export * from './user_action/latest';
 
 // V1
 export * as configureDomainV1 from './configure/v1';
+export * as userActionDomainV1 from './user_action/v1';
```

Comment:
I assume this is if we want to keep some API using an old version, we'll always be able to do `userActionDomainV*`.

But do we need to preemptively export them here? Feels a bit weird to be doing the same thing twice.

Reply:
> I assume this is if we want to keep some API using an old version, we'll always be able to do userActionDomainV*.
> But do we need to preemptively export them here? Feels a bit weird to be doing the same thing twice.

Not exactly. This is used to fix our routes to a specific version. If you see the routes, we use this type to fix the responses of the APIs. If something changes TS will inform us.